config/data/langs.toml

```diff
@@ -1423,6 +1423,29 @@ for arg in os.args[1..] {
 }
 '''
 
+[Vala]
+args       = [ '/usr/bin/valawrapper', '-' ]
+experiment = 1354
+size       = '194 MiB'
+version    = '0.56.17'
+website    = 'https://vala.dev'
+example    = '''
+int main(string[] args) {
```

Comment:
It'll be a cold day in hell when I return `void` from `main` for demonstration purposes.